pkgs/top-level/all-packages.nix

```diff
@@ -808,6 +808,8 @@ in
 
   dpt-rp1-py = callPackage ../tools/misc/dpt-rp1-py { };
 
+  ec2-instance-connect-cli = python3Packages.callPackage ../tools/security/ec2-instance-connect-cli { };
```

Comment:
There's a sort of agreement that using `callPackage` in `all-packages.nix` is better vs this method. This would require importing `python3Packages` inside the expression.